python/cuspatial/cuspatial/core/trajectory.py

```diff
@@ -92,35 +131,49 @@ def trajectory_bounding_boxes(num_trajectories, object_ids, xs, ys):
 def trajectory_distances_and_speeds(
     num_trajectories, object_ids, xs, ys, timestamps
 ):
-    """ Compute the distance traveled and speed of sets of trajectories
+    """
+    Compute the distance traveled and speed of sets of trajectories
 
     Parameters
     ----------
-    {params}
+    num_trajectories
+        number of trajectories (unique object ids)
+    object_ids
+        column of object (e.g., vehicle) ids
+    xs
+        column of x-coordinates (in kilometers)
+    ys
+        column of y-coordinates (in kilometers)
```

Comment:
The inputs must be in kilometers if meters/second is the desired output. Though if the inputs were in miles, the output speeds would be milli-miles/second since we're just dividing the input units 1000.

Reply:
Ah, I see. Perhaps We should leave the results in kilometers/s, or add an optional multiplier? Not relevant this PR, though.